From c5bdd39f8ebc4e6c58a47d7e424eac028eddb4ff Mon Sep 17 00:00:00 2001
From: Trevor Gamblin <tgamblin@baylibre.com>
Date: Fri, 15 Sep 2023 08:48:33 -0400
Subject: [PATCH] skip no_stdout_fileno test due to load variability

Skip test_input_no_stdout_fileno so that it doesn't fail on systems
under heavy load.

Upstream-Status: Inappropriate [OE-Specific]

[YOCTO #15210]

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 Lib/test/test_builtin.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Lib/test/test_builtin.py b/Lib/test/test_builtin.py
index c5394de..ed17fb6 100644
--- a/Lib/test/test_builtin.py
+++ b/Lib/test/test_builtin.py
@@ -2435,6 +2435,7 @@ class PtyTests(unittest.TestCase):
                          "byte 0xe9 in position 4: ordinal not in "
                          "range(128)")
 
+    @unittest.skip("Test may fail under heavy load")
     def test_input_no_stdout_fileno(self):
         # Issue #24402: If stdin is the original terminal but stdout.fileno()
         # fails, do not use the original stdout file descriptor
